src/chatEventWebhooks/chatAfterEvent/onMessageUpdated.js

```diff
@@ -16,10 +19,40 @@
  * DateCreated - date string - The timestamp of message creation
  * DateUpdated - date string - The timestamp of update of the message
  */
-const onMessageUpdated = (request, response) => {
-  return response
-    .status(200)
-    .json({});
+const onMessageUpdated = async (request, response) => {
+  try {
+    let pushStatus = {};
+    let { ChannelSid, Attributes = {}, From, ModifiedBy } = request.body;
+    const { consumers = [] } = Attributes;
+    if (consumers.includes(ModifiedBy)) {
```

Comment:
It should be "lastModifiedBy" according to their docs. 
http://media.twiliocdn.com/sdk/js/chat/releases/3.3.5/docs/Message.html

Reply:
Benji! This attribute is not passed to the webhook.
These are the attributes that I can get: https://www.twilio.com/docs/chat/webhook-events#onmessageupdated